src/chevah_compat/nt_filesystem.py

```diff
@@ -158,7 +158,13 @@ def installation_segments(self):
     def getEncodedPath(cls, path):
         '''Return the encoded representation of the path, use in the lower
         lever API for accessing the filesystem.'''
-        return path
+        if path.startswith(u'\\\\?\\'):
+            # This might be already and encoded long path.
+            return path
+
+        if len(path) < 250:
+            return path
+        return u'\\\\?\\' + path
```

Comment:
Is this the reason for this PR?
Maybe add a link to this long path format being documented:

```suggestion
        # An extended-length path, use the "\\?\" prefix.
        # https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation
        return u'\\\\?\\' + path
```